docs/pages/installation/eks.md

```diff
@@ -9,32 +9,41 @@ list: include
 image: /assets/img/platforms/eks.png
 ---
 
+Manage your EKS clusters with Meshery. Deploy Meshery on EKS or outside of EKS.
 {% include installation_prerequisites.html %}
 
-## Set up and run Meshery on EKS:
+### General Prerequisites:
+1. Access to an active EKS cluster in AWS Account.
```

Comment:
Do we know what specific level of credentials / authorization is needed? 

This info would be an improvement upon "in AWS Account".

Reply:
This can be improved as such like 1. Access to an active EKS cluster in any one of your region in your AWS Account.